### DISCUSSION TITLE
Welcome to tuliprox Discussions!

### DISCUSSION THREAD

**Discussion by euzu** [euzu/tuliprox#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hootie318** [comment]
Is there a wiki I can follow?

**Reply by euzu** [reply]
Sorry i didnt have time to write a wiki,  the only documentation is the README.md.
But feel free to ask if you have problems or want to understand the configuration.
There are example configuration files in the source folder.  `config.yml`, `mapping.yml` and `api-proxy.yml`

If you only want to run `m3u-filter`  for simple renaming and filtering you only need `config.yml`
If you want to use  advanced mapping features you need `mappings.yml`
If you want to run in server mode to serve the  created results you need `api-proxy.yml`

**Reply by euzu** [reply]
i have now copied the README.md content into the wiki. i'll be reviewing the texts when i have more time.
